src/Common/src/Interop/Windows/sspicli/SecuritySafeHandles.cs

```diff
@@ -872,7 +872,7 @@ public new IntPtr DangerousGetHandle()
                     GlobalLog.Assert("SafeDeleteContext::AcceptSecurityContext()|inSecBuffer == null || inSecBuffers == null");
                 }
 
-                Debug.Fail("SafeDeleteContext::AcceptSecurityContext()|outSecBuffer != null");
+                Debug.Fail("SafeDeleteContext::AcceptSecurityContext()|inSecBuffer == null || inSecBuffers == null");
```

Comment:
The if seems to want one (or both) to be null, so I'm not sure the assert string matches the intent?